### DISCUSSION TITLE
AI主播報新聞 70天無人發現

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#317] [閒聊八卦]
電視節目中出現AI主播，在中國已不止一次，但以往都是一看就能看出來。而每經AI電視於2021年10月11日上線測試，N小黑主打的財經脫口秀節目在抖音上有300萬粉絲，在70天中始終沒人看出來。
之前有看到同學分享AI寫新聞稿能騙過大家，現在甚至在新聞節目上播報也不會被發現XD
[https://www.epochtimes.com.tw/n365976/AI主播報新聞-70天無人發現.html](url)

**Comment by 109802503gloria** [comment]
自己查了很多相關影片來看，發現真的做得很細緻，不只臉部表情，連肢體動作都很自然

**Comment by ghost** [comment]
看到這個就連想到之前看到的新聞，雖然有點不太一樣
[你也被騙了四個月？解密NVIDIA執行長黃仁勳的「超級偽裝術」](https://www.gvm.com.tw/article/81716)

**Comment by cody910120** [comment]
現在科技真的跟鬼一樣ㄟ，我比較怕鬼之修圖

**Comment by 109302003** [comment]
這位ai主播的樣貌比ai女團自然、真實很多耶！

**Comment by Joan0518** [comment]
開始懷疑網路上看到的人物是真是假了...

**Comment by 107602505** [comment]
由AI當主播好像沒有甚麼缺點，甚至可以減少一些播報時的口誤等等，只是若發展起來未來主播的工作機會可能會變得很少QQ

**Comment by YuHengChen54** [comment]
現在網路上各種東西真的都不能信，太可怕了
